mlflow_kubernetes_plugins/auth/middleware.py

```
@@ -49,6 +61,53 @@ def __init__(self, app, authorizer, config_values):
         self.authorizer = authorizer
         self.config_values = config_values
 
+    @staticmethod
+    async def _parse_json_body(request: Request) -> object | None:
+        content_type = request.headers.get("content-type", "")
+        if "json" not in content_type.lower():
+            return None
+        try:
+            body = await request.body()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)